ci/deploy.sh

```diff
@@ -12,7 +12,8 @@ REPO_SLUG=${TRAVIS_REPO_SLUG:-$GITHUB_REPOSITORY}
 COMMIT=${TRAVIS_COMMIT:-$GITHUB_SHA}
 CI_BUILD_WEB_URL=${CI_BUILD_WEB_URL:-$TRAVIS_BUILD_WEB_URL}
 CI_JOB_WEB_URL=${CI_JOB_WEB_URL:-$TRAVIS_JOB_WEB_URL}
-BRANCH=${TRAVIS_BRANCH:-master}
+BRANCH=${TRAVIS_BRANCH:-$DEFAULT_BRANCH}
+BRANCH=${BRANCH:-$main}
```

Comment:
The approach looks good so far.  I can review when it's ready.

Did you want this here?
```suggestion
BRANCH=${BRANCH:-main}
```

Reply:
Nice catch. Feel free to review now. Will probably add something to the setup update instructions for updating users that still have a master default. They do not need to change, but the documentation will make more sense this way.